source/non_matching/coupling.cc

```diff
@@ -102,6 +121,22 @@ namespace NonMatching
       if (immersed_c[i])
         immersed_gtl[i] = j++;
 
+    // Construct a dof_mask, used to distribute entries to the sparsity
+    Table< 2, bool > dof_mask(space_fe.dofs_per_cell,
```

Comment:
i don't see where this is used in this function.

Reply:
You're right. It should have been used in add_local_to_gobal below, but I forgot to add it. 

Commit incoming, with test.